workers/data_refinery_workers/downloaders/utils.py

```diff
@@ -25,6 +28,7 @@
 SYSTEM_VERSION = get_env_variable("SYSTEM_VERSION")
 # TODO: extend this list.
 BLACKLISTED_EXTENSIONS = ["xml", "chp", "exp"]
+MAX_JOBS_PER_NODE = 15
```

Comment:
Two things about this. The first is that I think this is high if we're planning on scaling out to 20 instances. The second is that this should be an environment variable so that we can pass in whatever value we want to the docker container without having to rebuild it.

Reply:
Envariablized.